runtime/oti/j9nonbuilder.h

```diff
@@ -5443,6 +5446,9 @@ typedef struct J9JavaVM {
 	UDATA vmindexOffset;
 	UDATA vmtargetOffset;
 #endif /* defined(J9VM_OPT_OPENJDK_METHODHANDLE) */
+#if defined(J9VM_ZOS_3164_INTEROPERABILITY)
+	U_32 javaVM31;
```

Comment:
This will misalign the rest of the structure (once more is added).

Reply:
Added U_32 pad member immediately after.  https://github.com/eclipse/openj9/pull/12074/commits/1bb3113393e6c199034b4b023b886b6a656b2ee7